hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/staking/StakeRewardCalculatorImpl.java

```diff
@@ -40,10 +43,13 @@ public StakeRewardCalculatorImpl(@NonNull final StakePeriodManager stakePeriodMa
     /** {@inheritDoc} */
     @Override
     public long computePendingReward(
-            @NonNull final Account account,
+            @Nullable final Account account,
```

Comment:
Is it actually acceptable for client code to pass `null` here? (Not immediately obvious when that would happen as part of normal operations.)

Reply:
yes agreed. It wont be possible. Changed it.